test/integration/server.cc

```diff
@@ -21,6 +22,31 @@
 #include "gtest/gtest.h"
 
 namespace Envoy {
+namespace Server {
+
+OptionsImpl createTestOptionsImpl(const std::string& config_path, const std::string& config_yaml,
+                                  Network::Address::IpVersion ip_version) {
+  OptionsImpl test_options("cluster_name", "node_name", "zone_name", spdlog::level::info);
+
+  test_options.setBaseId(0u);
```

Comment:
Some of these are already set to the same values in the test constructor. What's the reasoning behind setting them again?